core/src/main/java/io/questdb/cutlass/http/processors/TextLoaderCompletedState.java

```diff
@@ -39,7 +39,7 @@ public void copyState(TextLoader textLoader) {
         // in order to release TableWriter back to the Engine
         this.writtenLineCount = textLoader.getWrittenLineCount();
         this.metadata = textLoader.getMetadata() != null
-                ? GenericRecordMetadata.copyOf(textLoader.getMetadata())
+                ? GenericRecordMetadata.cleanCopyOf(textLoader.getMetadata())
```

Comment:
I can change this to `GenericRecordMetadata.copyDense(textLoader.getMetadata())`
and all the tests still pass.